libraries/AP_HAL_ChibiOS/UARTDriver.h

```diff
@@ -133,11 +133,11 @@ class ChibiOS::UARTDriver : public AP_HAL::UARTDriver {
      */
     uint64_t receive_time_constraint_us(uint16_t nbytes) override;
 
-    uint32_t bw_in_kilobytes_per_second() const override {
+    uint32_t bw_in_bytes_per_second() const override {
         if (sdef.is_usb) {
-            return 200;
+            return 200*1024;
         }
-        return _baudrate/(9*1024);
+        return _baudrate/10;
```

Comment:
Why change from 9 to 10 here?

Pretty sure the 9 reflects `8N1`

Reply:
1 start byte + 8 data bytes +  1 stop byte ...